app/scripts/controllers/preferences.js

```diff
@@ -14,6 +14,14 @@ class PreferencesController {
   }
 // PUBLIC METHODS
 
+  setUseBlockie (val) {
+    this.store.updateState({ useBlockie: val })
```

Comment:
Should also include a "default value" for this key in the constructor.

Reply:
I'm probably just being stupid, but I'm not sure what you mean. The useBlockie property is initialized with a false value by the metamask.js state before Redux syncs the prop values up, is this not sufficient?